WalletWasabi.Documentation/Guides/InstallInstructions.md

```diff
@@ -110,58 +113,58 @@ sudo gpg2 --verify {file name}.asc {file name}.dmg
 
-1. Download Gpg4win from https://www.gnupg.org/download/index.html
-2. Install Gpg4win 
+1. Download Gpg4win from https://www.gnupg.org/download/index.html.
+2. Install Gpg4win.
 
 ![Install Gpg4win](https://i.imgur.com/YKDdw1k.png)
 
-3. Download Wasabi latest __release__ and the corresponding __.asc__ file.
+3. Download Wasabi's latest __release__ and the corresponding __.asc__ file.
 
-4. Double click on .asc file or right click More GpgEX options / Verify. If the context menu is missing - restart the computer.
+4. Double click on .asc file or right click it and navigate to More GpgEX options and click Verify. If the context menu is missing restart the computer.
 
 ![](https://i.imgur.com/fJME8Yh.png)
 
 5. Press Search.
 
 ![](https://i.imgur.com/cj00rev.png)
 
-6. Wait until Ficsór Ádám's certificate (adam.ficsor73@gmail.com) appears. Who is this guy? The owner of WasabiWallet AKA [nopara73]( https://github.com/nopara73)
+6. Wait until Ficsór Ádám's certificate (adam.ficsor73@gmail.com) appears. Who is this guy? The owner of WasabiWallet AKA [nopara73](https://github.com/nopara73).
 
 ![](https://i.imgur.com/B3WZn1n.png)
 
-7. Add Ádám's certificate. (On the next release you can skip previous steps because the certificate will be there).
+7. Add Ádám's certificate. (Next time you want to verify a new release you can skip the previous steps because the certificate will be there).
 
 ![](https://i.imgur.com/9zGpuI6.png)
 
-8. Select all and verify the fingerprint: `21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`
+8. Select all and verify the fingerprint: `21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`.
 
 ![](https://i.imgur.com/PfdbegY.png)
 
-9. Press next, next, next. If there is an error, you can try to import the key manually, navigate to section: [Import key manually](https://github.com/molnard/WalletWasabi/blob/patch-3/WalletWasabi.Documentation/Guides/InstallInstructions.md#import-key-manually)
+9. Press next, next, next. If there is an error, you can try to import the key manually, navigate to section: [Import key manually](https://github.com/molnard/WalletWasabi/blob/patch-3/WalletWasabi.Documentation/Guides/InstallInstructions.md#import-key-manually).
 
-9. Successful validation. The file was signed by the developer.
+10. Successful validation. The file was signed by the developer.
 
 ![Imgur](https://i.imgur.com/7e0O9dQ.png)
 
-10. You can install Wasabi with the msi.
+11. You can install Wasabi with the msi.
 
 ### Import key manually
 
-1. Open this site, you will find the developer's (nopara73) public key there [nopara73's PGP](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt). Press Copy.
+1. Go to [this link](https://github.com/zkSNACKs/WalletWasabi), you will find the developer's (nopara73) [public key](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt) there. Press Copy.
 
 ![Imgur](https://i.imgur.com/zLVqhOu.png)
 
-2. Create a TXT file pgp.txt.
+2. Create a TXT file and name it pgp.
```

Comment:
Nit:
Should be either "pgp" or `pgp`

Reply:
On the document it is "pgp". `pgp` is only on github.